### ISSUE TITLE
Replace splash page with expanded notice

### ISSUE THREAD

**Issue by taisdesouzalessa** [mozilla/network#519]
**Expanded Copy:**
DESKTOP
This is a Barn Raising
There is a movement to keep the Internet healthy taking root around the world. Mozilla is a part of this movement and wants to help it grow.

We're building a home for people who care about the health of the Internet, hand in hand with the community that emerged from MozFest and events around the world. It’s a space for us to learn, find resources, and connect to new people and ideas.

If you are a part of this movement for Internet Health then we want your [feedback ](https://mzl.la/2ohCL8O)to help it grow. This is a barn raising.

MOBILE
This is a Movement
We're building a home for people who care about the health of the Internet. This is a space for us to learn, find resources, and connect. If you're a part of the movement for Internet Health, we want your [feedback](https://mzl.la/2ohCL8O) and help.

**Minified Copy:**
Welcome to our member preview. It’s a work in progress and we invite your [feedback](https://mzl.la/2ohCL8O).
_____
**Mockup:**
Desktop: https://invis.io/NHC4H7MEM#/238506265_Desktop-Expanded-2col
Mobile: https://invis.io/GKC4CTZN4#/238463781_Mobile-Expanded
___
**Animation:**
![banner-v4](https://user-images.githubusercontent.com/22157921/27109560-9ab1c61a-5058-11e7-98ae-cb407abe14b3.gif)
____
**Sketch file:**
https://drive.google.com/a/mozilla.com/file/d/0B3b1KWRwjxtoLWhxMWtxNGd5cHM/view?usp=sharing

**Comment by hannahkane** [comment]
estimating as written, though we know some changes might be requested
MVP for animation is: Slightly Janky

**Comment by gvn** [comment]
@taisdesouzalessa Can you attach the SVG for the **+** ?

**Comment by taisdesouzalessa** [comment]
hey @gvn, here is the svg for the "+". It is the same as the "x", but rotated 45°. I used the same asset as our menu "x" - but smaller.

I can't upload svg here, so I compressed it to zip - let me know if you have any tricks to get straight to svg here, maybe I am missing something?
[extended-plus-top-banner-optimised.svg.zip](https://github.com/mozilla/network/files/1139278/extended-plus-top-banner-optimised.svg.zip)
